Xamarin.PropertyEditing.Mac/Controls/CGRectEditorControl.cs

```diff
@@ -48,6 +48,8 @@ protected override void UpdateModelValue ()
 			YEditor.StringValue = ViewModel.Value.Y.ToString ();
 			WidthEditor.StringValue = ViewModel.Value.Width.ToString ();
 			HeightEditor.StringValue = ViewModel.Value.Height.ToString ();
+
+			ViewModel.RowHeight = 48;
```

Comment:
@ermau The Heights are being set by the View. The VM is being passed to the GetRowHeight method via the NSObjectFacade. Unless we also try to pass the view as another property of the NSObjectFacade.